src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,12 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public static String getPrintableString(Printable ...printables) {
+        String results = new String();
+        for(Printable i:printables) {
+            results.concat(i.getPrintableString()+" ");
```

Comment:
Can be more efficient by using the `StringBuilder` class which helps join strings.